Assets/Scripts/ssjAug2018/DeilveryTarget/DeliveryTargetManager.cs

```diff
@@ -0,0 +1,9 @@
+﻿using pdxpartyparrot.Core.Actors;
+using pdxpartyparrot.ssjAug2018.DeliveryTargets;
+
+using UnityEngine;
+
+public class DeliveryTargetManager : ActorManager<DeliveryTarget> {
```

Comment:
this shouldn't be an Actor manager, you can just make it a SingletonBehavior, similar to SpawnpointManager